src/Microsoft.Identity.Web/WebAppExtensions/MicrosoftIdentityWebAppAuthenticationBuilder.cs

```diff
@@ -55,14 +55,9 @@ public class MicrosoftIdentityWebAppAuthenticationBuilder : MicrosoftIdentityBas
         /// <param name="initialScopes">Initial scopes.</param>
         /// <returns>The builder itself for chaining.</returns>
         public MicrosoftIdentityAppCallsWebApiAuthenticationBuilder EnableTokenAcquisitionToCallDownstreamApi(
```

Comment:
We'd also want to have an override:

public `MicrosoftIdentityAppCallsWebApiAuthenticationBuilder EnableTokenAcquisitionToCallDownstreamApi(            IEnumerable<string>? initialScopes)`, which would call this method passing null as the first parameter. This would improve the DevEx as today customers will need to call `EnableTokenAcquisitionToCallDownstreamApi(null, new string[]{scopes})`, which is not totally natural

Reply:
ok. makes sense.

---
In reply to: [567938230](https://github.com/AzureAD/microsoft-identity-web/pull/910#discussion_r567938230) [](ancestors = 567938230)